src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -859,11 +868,18 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 			onDismiss: { [weak self] in
 				if self?.model.coronaTest?.positiveTestResultWasShown == true {
 					self?.showSkipCheckinsAlert(dontShareHandler: {
-						if let coronaTestType = self?.model.coronaTestType {
-							Analytics.collect(.keySubmissionMetadata(.submittedAfterCancel(true, coronaTestType)))
-							self?.submitExposure(showSubmissionSuccess: false) { isLoading in
-								footerViewModel.setLoadingIndicator(isLoading, disable: isLoading, button: .secondary)
-								footerViewModel.setLoadingIndicator(false, disable: isLoading, button: .primary)
+						if isSRSFlow {
+							self?.showSRSFlowAlert(
+								for: .consent(.cancelWarnOthers(on: checkinsVC)),
+								isLoading: { _ in }
+							)
+						} else {
+							if let coronaTestType = self?.model.coronaTestType {
+								Analytics.collect(.keySubmissionMetadata(.submittedAfterCancel(true, coronaTestType)))
+								self?.submitExposure(showSubmissionSuccess: false) { isLoading in
+									footerViewModel.setLoadingIndicator(isLoading, disable: isLoading, button: .secondary)
+									footerViewModel.setLoadingIndicator(false, disable: isLoading, button: .primary)
+								}
```

Comment:
The "change" just comes from a code movement into the new else block `if isSRSFlow { ... } else { ... }`

Reply:
Yep, that change wasn't introduced in this PR, but the code was moved inside the if-scope with the SRS changes, I mentioned it here in the story PR, that's why I'm suggesting it could be reverted:
https://github.com/corona-warn-app/cwa-app-ios/pull/4891#discussion_r1030845569